app/templates/main/volunteerProfile.html

```diff
@@ -191,9 +191,9 @@ <h5 class="modal-title">Ban Volunteer</h5>
           <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
         </div>
         <div class="modal-body" style="align=center">
-          <h6 align="left">{{volunteer.firstName}} {{volunteer.lastName}}</h6>
-          <h6 align="left">{{volunteer.email}}</h6>
-          <h6 align="left"><a href="tel:{{volunteer.phoneNumber}}">{{volunteer.phoneNumber}}</a></h6>
+          <h6 align="left" id="fullName">Full Name: {{volunteer.firstName}} {{volunteer.lastName}}</h6>
+          <h6 align="left" id="email">Email Address: {{volunteer.email}}</h6>
+          <h6 align="left" id="phoneNumber">Phone Number: <a href="tel:{{volunteer.phoneNumber}}">{{volunteer.phoneNumber}}</a></h6>
```

Comment:
Are these elements referenced by javascript? If not, we don't need `id` attributes for them.

Reply:
I'm not referencing the elements by javascript, so I removed the id attributes for them